spec/factories/tasks.rb

```diff
@@ -5,5 +5,12 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :complete_task do
+      status { 'done' }
```

Comment:
enumの値はSymbol形式で記載しましょう。ただの文字列でないことが伝わります。
```suggestion
      status { :done }
```

Reply:
aec2798